packages/mobile/src/screens/sign-in-screen/SignInScreen.tsx

```diff
@@ -50,6 +51,14 @@ export const SignInScreen = () => {
               icon={IconArrow}
               onPress={() => handleSubmit()}
             />
+            <Link
+              to={{
```

Comment:
all good, but with small objects i like single line

Reply:
Fair, but tbh I'm surprised that prettier didn't automatically do it for me 🤔